src/window.rs

```diff
@@ -821,14 +821,11 @@ impl Window {
         self.window.is_decorated()
     }
 
-    /// Change whether or not the window will always be on top of other windows.
-    ///
-    /// ## Platform-specific
+    /// Change the window level.
     ///
```

Comment:
Could you add a note that it's just a hint and the system could ignore the request? Could put it on a enum variant.